package.json

```diff
@@ -1,17 +1,21 @@
 {
   "name": "@conveyal/lonlat",
   "description": "Lon/lat normalization",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "module": "dist/lonlat.esm.js",
+  "typings": "dist/index.d.ts",
   "scripts": {
-    "cover": "yarn test --coverage --coverage-paths index.js",
-    "generate-docs": "documentation readme index.js --section=API --markdown-toc=true",
-    "jest": "mastarm test",
-    "lint": "mastarm lint index.js index.test.js",
-    "lint-docs": "documentation lint index.js",
+    "build": "tsdx build -i index.ts --target node",
+    "cover": "yarn jest --coverage --collectCoverageFrom=index.ts",
+    "generate-docs": "documentation readme index.ts --parse-extension ts --section=API --markdown-toc=true",
+    "jest": "tsdx test",
+    "lint": "tsdx lint index.ts index.test.ts",
+    "lint-docs": "documentation lint index.ts",
+    "prepare": "tsdx build -i index.ts --target node",
     "pretest": "yarn",
     "semantic-release": "semantic-release",
-    "test": "yarn run lint && yarn run lint-docs && yarn run jest",
-    "travis-deploy-once": "travis-deploy-once"
+    "test": "yarn run jest && yarn run lint-docs",
```

Comment:
The previous `test` command should still be used here to make sure linting is included.